docs/book/src/development/development.md

```diff
@@ -342,6 +342,86 @@ $ sudo ln -s /run/user/$(id -u)/podman/podman.sock /var/run/docker.sock
 [hack-ci-gce-project]: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/hack/ci/gce-project.sh
 [hack-ci-openstack]: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/hack/ci/openstack.sh
 
+## Cluster Creation with ClusterClass
```

Comment:
Could you move this whole section up to https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/e89ec9ce14067866bab80c84da5060980a378f36/docs/book/src/development/development.md?plain=1#L52 ?
```suggestion
## Test Cluster Creation with the dev-test ClusterClass
```